src/VirtoCommerce.PricingModule.Web/Module.cs

```diff
@@ -88,16 +89,25 @@ public void Initialize(IServiceCollection serviceCollection)
                 configure.AddPolicy(typeof(ExportablePrice).FullName + "ExportDataPolicy", exportPolicy);
                 configure.AddPolicy(typeof(ExportablePricelistAssignment).FullName + "ExportDataPolicy", exportPolicy);
             });
+
+
+            var snapshot = serviceCollection.BuildServiceProvider();
+            var configuration = snapshot.GetService<IConfiguration>();
+            serviceCollection.AddOptions<SimpleExportOptions>().Bind(configuration.GetSection("Pricing:SimpleExport")).ValidateDataAnnotations();
```

Comment:
Need to avoid calling "serviceCollection.BuildServiceProvider()". Possible workaround: https://github.com/VirtoCommerce/vc-module-order/blob/dev/src/VirtoCommerce.OrdersModule.Web/Module.cs#L70

Reply:
a better workaround: extend your Module.cs with IHasConfiguration and use Configuration property
an example:
https://github.com/VirtoCommerce/vc-module-elastic-search/blob/dev/src/VirtoCommerce.ElasticSearchModule.Web/Module.cs

Reply:
@ksavosteev  What version of the platform did the interface appear in?

Reply:
@trueboroda 3.48.0

Reply:
@eMazeika  have suggested the best option how to avoid building a snapshot

Reply:
@tatarincev We already implemented a solution suggested by @ksavosteev. We think is the best.